### PR TITLE
README.md: document cork a little, give a nice example of building CoreOS

### PR DESCRIPTION
**Pull request by mischief**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,3 +119,65 @@ used to tear down the cluster. Common usage:
...
+docker run --rm -v "$PWD":/usr/src/myapp -w /usr/src/myapp golang ./build
+cp bin/* $HOME/bin/
+
+mkdir $HOME/coreos
+cd $HOME/coreos
+
+# download CoreOS SDK, build packages and image
+cork create --replace --sdk-version=${V} --verbose
+cork enter -- ${S}/update_chroot --toolchain_boards=${B}
+cork enter -- ${S}/build_packages --board=${B} --skip_chroot_upgrade --getbinpkgver=master --toolchainpkgonly
```
Comment (marineam) [inline README.md]: This is kinda specific to our own automated builds, except for building our master image I wouldn't recommend using binary packages from the master build. Just drop `--skip_chroot_upgrade --getbinpkgver=master --toolchainpkgonly` and the `update_chroot` command

**Comment by mischief** [comment]
@marineam ptal, updated per your commands. the command to run tests should work now that coreos/coreos-overlay#1713 landed.

**Comment by mischief** [comment]
although, should i bother calling `set_shared_user_password.sh` and `setup_board` here too?

**Comment by marineam** [comment]
LGTM, and no need to call other commands, this doesn't need to cover the SDK in general, just cork.

**Pull request merged**